infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -170,7 +175,7 @@
         bootstrapEndpoint = (opts.get("--management-server") != null)
                 ? opts.get("--management-server").toString() : null;
 
-        sequencerBootstrappedFuture = new CompletableFuture<>();
+        recoveryBarrierFuture = new CompletableFuture<>();
```

Comment:
Isn't this redundant ?

bootstrapPrimarySequencerServer will never execute before recoveryBarrierFuture is done (i.e. it will keep looping). Also, triggerSequencerBootstrap doesn't depend on its value, why have it ?

Reply:
I did attempt to remove it but was affecting a lot of tests. And I did not want to change any functionality. I just changed the name so that it is a little different and intuitive compared to the new CompletableFuture introduced - `sequencerRecoveryFuture`. I can do this with the cleanup we discussed.